src/back/spv/writer.rs

```diff
@@ -1311,6 +1309,11 @@ impl Writer {
         self.logical_layout.in_words(words);
         Ok(())
     }
+
+    /// Return the set of capabilities the last module written used.
+    pub fn get_capabilities_used(&self) -> crate::FastHashSet<spirv::Capability> {
```

Comment:
why is this not returning a reference? if the client wants to clone, they can do it